com.mojang.Minecraft.json

```diff
@@ -32,9 +19,6 @@
         "--share=network",
         "--env=JAVA_HOME=/app/jre"
```

Comment:
Is this still relevant?

Reply:
Nope! I just removed it locally after I saw it